tests/Library/Vanilla/Site/SingleSiteSectionProvider.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @author Adam Charron <adam.c@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\Library\Vanilla\Site;
+
+use Vanilla\Site\DefaultSiteSection;
+use Vanilla\Site\SingleSiteSectionProvider;
+use VanillaTests\MinimalContainerTestCase;
+
+/**
+ * Tests for SingleSiteSectionProvider.
+ */
+class SingleSiteSectionProviderTests extends MinimalContainerTestCase {
+
+    const LOCALE_KEY = 'AB-df';
```

Comment:
What does `AB-df` mean? I expect either `en_US` or `en-us`.
We should try to keep test data to contain pretty explicit and valid values.
If you want it to be different from default one we can use `fr` or `es` variants

Reply:
I had chosen this as it's a value that will never be set on any actual system ever. Would you prefer something like `fake-locale`?

Reply:
Adjusted to be `en_US`